src/components/stepper/stepper.jsx

```diff
@@ -51,7 +54,7 @@ module.exports = React.createClass({
           <Button handleClick={this.decrement} type="button" disabled={!this.canDecrement()}>{this.props.decrementDisplayString}</Button>
         </span>
 
-        <Input type="text" key={this.props.value} readOnly={true}>{this.props.value.toString()}</Input>
+        <Input type="text" id={this.props.id} key={this.props.value} readOnly={true} label={this.props.label}>{this.props.value.toString()}</Input>
```

Comment:
Not sure label is used right here. I was expecting a label to be passed into the stepper component and a `<label for="#">` to be put in front of the stepper. Is this right?